projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -13,6 +13,14 @@
  * Only user facing pieces of Publicize are found here.
  */
 class Publicize_UI {
+	/**
+	 * If Publicize is executing within Jetpack.
+	 *
+	 * @access public
+	 * @var bool
+	 */
+	public $in_jetpack;
```

Comment:
I wonder if this is actually needed. The one call I see setting this is at https://github.com/Automattic/jetpack/blob/9a37cb2ded4385c0294a26759e52be395771cdb0/projects/plugins/jetpack/modules/publicize.php#L108 but I don't see anything anywhere actually using the value anymore (the one historical use [appears to have been removed in #23786 a year ago](https://github.com/Automattic/jetpack/pull/23786/files#r849322516)).

Reply:
That's a good point. I removed it in 599862fcada2f5319934bc7b7e6725d285c7ab50